app/settings/TestRouteContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TestRouteContainer = props => {
+    console.log(props);
+    return (
+        <div><main role="main">
+            <h1>Hello from react</h1>
+            { props.id }
+            <a onClick={ () =>  { props.state.go('posts.data') } }>Click me</a>
+            <a href="/views/map">Click me</a>
```

Comment:
It seems a simple href works just as well as using `state.go` so I'd probably stick with hrefs for simpler refactoring in future.